ports/sonic.mania/sonicmania/BUILDING.md

```diff
@@ -0,0 +1,27 @@
+## Building Sonic Mania & RSDKv5
+This guide assumes you will be using WSL2 or similar with debian bullseye chroot.
```

Comment:
Maybe a link here to the RSDKv5 repo and something about the plus version not allowed to be distributed? 😄